agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.cpp

```diff
@@ -1480,6 +1483,88 @@ bool monitor_thread::handle_ap_metrics_query(Socket &sd, ieee1905_1::CmduMessage
     return message_com::send_cmdu(slave_socket, cmdu_tx);
 }
 
+bool monitor_thread::handle_beacon_metrics_query(Socket &sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    const auto mid = cmdu_rx.getMessageId();
+    int dialog_token;
```

Comment:
if this is used only to create the request at the end to the function, should be moved closer to point of use